build.sbt

```diff
@@ -313,8 +311,6 @@ val mtagsSettings = List(
     (ThisBuild / baseDirectory).value / "mtags",
     scalaVersion.value
   ),
-  // @note needed to deal with issues with dottyDoc
-  Compile / doc / sources := Seq.empty,
```

Comment:
If I'm not mistaken this should work now right? Or is it silly to remove since we don't use the docs anyways?

Reply:
That was removed mostly due to issues with the Dotty plugin, it's fine to remove if everything publishes fine for the different Scala 3 versions. Might be useful to run `+mtags/publishLocal` to test that out.

Reply:
Ok perfect. I did a full `+publishLocal` to test everything so this should be good then 👍🏼

Reply:
What the heck... breaks in CI.

Reply:
Hmm... This might not be related, it seems we are failing to resolve from bintray, which is shutting down I think? I am not sure though  why it's not available on maven central then.

Reply:
```
We replaced the isDottyCompat extension method by the static for3Use2_13 object whose usage is:

(“a” % “b” % “1.0”).cross(CrossVersion.for3Use2_13)

Symmetrically we added the for2_13Use3 that has no counterpart in sbt-dotty.

For now, the doc task is made available on Scala 3 by setting:

thisBuild / resolvers += Resolver.JCenterRepository

This is due to scala3doc depending on Dokka, whose artifacts are not available in the default resolvers. This dependency is planned to be removed.
```

from https://contributors.scala-lang.org/t/scala-center-q4-2020-update/4873

Reply:
That is not related, but this is https://github.com/sbt/sbt/issues/6294#issuecomment-817836965 :/

Reply:
Yeah, I think this might not be fixed for M1, we would need to drop it.

Reply:
> Yeah, I think this might not be fixed for M1, we would need to drop it.

Sounds good. For now I kept it in, and then when we drop M1 we can remove it. So nice to see all ✅ again.